butia_emotions:

  eyebrown:

    standard:
      rightY: 50
      leftY: 50
      rightRotation: 50
      leftRotation: 50
      
    happy:
      rightY: 90
      leftY: 90
      rightRotation: 60
      leftRotation: 60

    sad:
      rightY: 30
      leftY: 30
      rightRotation: 20
      leftRotation: 20

    rage:
      rightY: 10
      leftY: 10
      rightRotation: 80
      leftRotation: 80

    scared:
      rightY: 100
      leftY: 100
      rightRotation: 70
      leftRotation: 70

  eyelid:

    standard:
      h: 60
      frequency: 2

    happy:
      h: 80
      frequency: 3

    sad:
      h: 20
      frequency: 4

    rage:
      h: 10
      frequency: 3

    scared:
      h: 90
      frequency: 1